ifermi/plot.py

```diff
@@ -279,6 +280,8 @@ def get_plot(
                 combination with the ``vector_properties`` option.
             hide_labels: Whether to show the high-symmetry k-point labels.
             hide_cell: Whether to show the reciprocal cell boundary.
+            plot_index: The indices of the surfaces to plot. If none are input then the
+                entire surface will be plotted.
```

Comment:
Can you update this docstring with the additional information you added to `all_properties` etc.